### DISCUSSION TITLE
Google Colab notebook not working

### DISCUSSION THREAD

**Discussion by SebastianMiles** [RosettaCommons/RFdiffusion#173] [General]
Hi, i've been using the colab notebook for a couple of weeks now, but since the last Thursday it doesn't work anymore

There are several errors when running the setup

Thanks in advance

`---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
<timed exec> in <module>

[/content/RFdiffusion/inference/utils.py](https://localhost:8080/#) in <module>
      6 import torch
      7 import torch.nn.functional as nn
----> 8 import diffusion
      9 from diffusion import get_beta_schedule
     10 from scipy.spatial.transform import Rotation as scipy_R

7 frames
[/content/RFdiffusion/diffusion.py](https://localhost:8080/#) in <module>
     15 from util import reference_angles as REF_ANGLES
     16 
---> 17 from util_module import ComputeAllAtomCoords
     18 
     19 from diff_util import th_min_angle, th_interpolate_angles

[/content/RFdiffusion/util_module.py](https://localhost:8080/#) in <module>
      5 from opt_einsum import contract as einsum
      6 import copy
----> 7 import dgl
      8 from util import base_indices, RTs_by_torsion, xyzs_in_base_frame, rigid_from_3_points
      9 

[/usr/local/lib/python3.10/dist-packages/dgl/__init__.py](https://localhost:8080/#) in <module>
     16 
     17 # Should import backend before importing anything else
---> 18 from .backend import load_backend, backend_name
     19 
     20 from . import function

[/usr/local/lib/python3.10/dist-packages/dgl/backend/__init__.py](https://localhost:8080/#) in <module>
    120 
    121 
--> 122 load_backend(get_preferred_backend())
    123 
    124 

[/usr/local/lib/python3.10/dist-packages/dgl/backend/__init__.py](https://localhost:8080/#) in load_backend(mod_name)
     49         raise NotImplementedError("Unsupported backend: %s" % mod_name)
     50 
---> 51     from .._ffi.base import load_tensor_adapter  # imports DGL C library
     52 
     53     version = mod.__version__

[/usr/local/lib/python3.10/dist-packages/dgl/_ffi/base.py](https://localhost:8080/#) in <module>
     48 __version__ = libinfo.__version__
     49 # library instance of nnvm
---> 50 _LIB, _LIB_NAME, _DIR_NAME = _load_lib()
     51 
     52 # The FFI mode of DGL

[/usr/local/lib/python3.10/dist-packages/dgl/_ffi/base.py](https://localhost:8080/#) in _load_lib()
     37     """Load libary by searching possible path."""
     38     lib_path = libinfo.find_lib_path()
---> 39     lib = ctypes.CDLL(lib_path[0])
     40     dirname = os.path.dirname(lib_path[0])
     41     basename = os.path.basename(lib_path[0])

[/usr/lib/python3.10/ctypes/__init__.py](https://localhost:8080/#) in __init__(self, name, mode, handle, use_errno, use_last_error, winmode)
    372         self._FuncPtr = _FuncPtr
    373 
--> 374         if handle is None:
    375             self._handle = _dlopen(self._name, mode)
    376         else:

OSError: libcudart.so.11.0: cannot open shared object file: No such file or directory`

**Comment by xinran1129305** [comment]
Hi there. How did you solve this problem?

**Comment by roccomoretti** [comment]
Cross referencing from issue #174 -- Google colab updated the CUDA version, so the notebook version v1.1.1 no longer works, and you need to update to the [latest version](https://colab.research.google.com/github/sokrypton/ColabDesign/blob/main/rf/examples/diffusion.ipynb)
